10.9-libcxx/stable/main/finkinfo/sci/qhull6-shlibs.info

```diff
@@ -31,11 +36,10 @@ Source: http://www.qhull.org/download/qhull-%v-src.tgz
 Source-MD5: 4f4bac7769a5b1e0c6470ea457e2d4d2
 SourceDirectory: qhull-%v
 
-Depends: %N-shlibs (= %v-%r )
-Conflicts: qhull, qhull5, qhull6.3, qhull6.3.1
-Replaces: qhull, qhull5, qhull6.3, qhull6.3.1
+Conflicts: qhull6-dev (<< 2011.2-5) 
+Replaces: qhull6-dev (<< 2011.2-5) 
 
-PatchFile: %n.patch
+PatchFile: qhull6.patch
```

Comment:
Probably best to keep the .patch in sync with the .info

Reply:
Done